wallet-connect/.eslintrc.js

```diff
@@ -0,0 +1,53 @@
+module.exports = {
+  env: {
+    browser: true,
+    node: true,
+    es6: true,
+    es2020: true,
+    mocha: true
+  },
+  plugins: ["simple-import-sort", "sonarjs"],
+  extends: [
+    "plugin:sonarjs/recommended",
+    "eslint:recommended",
+    "plugin:react/recommended"
```

Comment:
this is not a react project. Where did you copy this file from?

Reply:
Converted to vue